constants/map_constants.asm

```diff
@@ -292,5 +298,9 @@ ENDM
 	map_const OFFICE_3,                         10, 18 ; 16
 	map_const SLOWPOKE_WELL_ENTRANCE,           10,  9 ; 17
 	map_const SLOWPOKE_WELL_MAIN,               10, 18 ; 18
+	endgroup
+
+	newgroup  EMPTY                                    ; 17
+	endgroup
```

Comment:
`data/maps/maps.asm` defines an empty map group, so I'd thought I add it here as well